docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -91,6 +100,7 @@ This method of provisioning the cluster would act as a single control point for
 - Extend the Cluster object to use ClusterClass for creating managed topologies.
 - Enhance the Cluster object to act as a single point of control for the topology.
 - Extend the Cluster controller to create/update/delete managed topologies.
+- Introduce mechanisms for allowing Cluster-specific transformations of a ClusterClass, and the continuous reconciliation of the Cluster managed resources.
```

Comment:
```suggestion
- Introduce mechanisms for allowing Cluster-specific transformations of a ClusterClass, and the continuous reconciliation of the ClusterClass managed resources.
```
Is this more correct? Or Cluster's managed resources?

Reply:
Hm maybe it's rather `of the topology managed resources`

Reply:
Yeah - that works I think.